tests/containers_conf.bats

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "containers.conf env test" {
+    export CONTAINERS_CONF=${TESTSDIR}/containers.conf 
+    cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json docker.io/alpine)
+    run_buildah --log-level=error run $cid sh -c 'env | grep "foo=bar"'
+}
+
+@test "containers.conf selinux test" {
+    if ! which selinuxenabled > /dev/null 2> /dev/null ; then
+        skip "No selinuxenabled"
```

Comment:
s/selinuxenabled/selinux enabled/

Reply:
That is saying the selinuxenabled program does not exists.  So it should stay as a single name.  I will add "No selinuxenabled application"